sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_sender_async.py

```diff
@@ -171,6 +172,7 @@ def _create_handler(self, auth):
             client_name=self._name,
             keep_alive_interval=self._config.keep_alive,
             encoding=self._config.encoding,
+            link_properties = {CONSUMER_IDENTIFIER:self._name}
```

Comment:
same question about why consumer identifier here instead of producer identifier?